ArticleTemplates/assets/scss/modules/_resizable.scss

```diff
@@ -19,3 +19,11 @@
         }
     }
 }
+
+.resizable_line_height {
+    @for $i from 0 through 10 {
+        .line-height-#{$i} & {
+            line-height: 1.5 + ($i * 0.1);
```

Comment:
Does this preserve the existing default line height?

Reply:
As I understand it, this should preserve the default line height at 1.5.